cli/azd/pkg/project/service_target_containerapp.go

```diff
@@ -42,7 +43,7 @@ func (at *containerAppTarget) Deploy(ctx context.Context, azdCtx *azdcontext.Azd
 		at.config.Infra.Module = at.config.Name
 	}
 
-	infraManager, err := provisioning.NewManager(ctx, *at.env, at.config.Project.Path, at.config.Infra, false)
+	infraManager, err := provisioning.NewManager(ctx, *at.env, at.config.Project.Path, at.config.Infra, true)
```

Comment:
This hardcoded value makes me wonder if the `interactive` options isn't passed correctly, or that provisioning manager shouldn't know about interactivity if it's meant to be used in this way..